content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+
+The expected values are:
+- `field`: the attribute code
+- `value`: the attribute value
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+
+#### Example
+
+To add the “t-shirts” category, the action will be as follows:
+
+```YML
+  actions:
+      type: add
+        field: categories
+        items:
+          - t-shirts
+```
+
+### Set
+
+The `Set` action assigns value(s) to every attributes, categories, family, associations, groups, the "parent" field, or the "enabled" field.
+
+The expected values are:
+- `field`: the attribute code or category code or family code
+- `value`: the attribute value or category code or family code
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+
+#### Example
+
+To set the *“My very new description for purple T-shirt”* value to your `description` attribute in the `en_US` locale and for the `e-commerce` channel, the action will be as follows:
+
+```YML
+  actions:
+      type:   set
+        field:  description
+        locale: en_US
+        scope:  ecommerce
+        value:  "My very new description for purple tshirt"
+```
+
+## Remove
+
+The `Remove` action enables you to remove values from a multi-select attribute, a reference entity multiple link attribute or a product category.
+
+The expected values are:
+- `field`: the attribute code or “categories”
+- `items`: the values to remove
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+- `include_children`: if true, then it also applies the removal of the children to the given categories. It is only applicable if the field is set to “categories” (It's optional. By default, it is set to false)
+
+
+### Example
+
+To remove the “t-shirts” category, the action will be as follows:
+
+```YML
+  actions:
+      type: remove
+        field: categories
+        items:
+          - t-shirts
+```
+
+To remove the “clothing” category and its children, the action will be as follows:
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - clothing
+      include_children: true
+```
+
+To unclassify products from the whole “Master catalog” tree, the action will be as follows:
+
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - master
+      include_children: true
+```
+
+## Concatenate
+
+The `Concatenate` action concatenates at least two values into a single value. A space separates each source value.
+
+The possible source attribute types are:
+ - text
+ - text area
+ - date
+ - identifier
+ - metric
+ - number
+ - price collection
+ - simple or multi select (option codes)
+
+The possible target attribute types are:
+ - text
+ - text area
```

Comment:
I think it's "textarea"

Reply:
in the UI, it's "text area". Don't know if it's better to use the UI wording or the technical one. Same for "price". In the UI, it's not "price collection", but "price".